src/GraphCanvas.tsx

```diff
@@ -120,7 +121,14 @@ export const GraphCanvas: FC<GraphCanvasProps & { ref?: Ref<GraphCanvasRef> }> =
       const canvasRef = useRef<HTMLCanvasElement | null>(null);
 
       useImperativeHandle(ref, () => ({
-        centerGraph: (n?: string[]) => rendererRef.current?.centerGraph(n),
+        centerGraph: ({
```

Comment:
I probably would have made nodes the first arg and then all the others a options arg. I think this is fine for now though.